easybuild/easyconfigs/s/scVelo/scVelo-0.3.1-foss-2023a.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'scVelo'
+version = '0.3.1'
+local_commit = 'ffe2cd3'
+
+homepage = "https://scvelo.org"
+description = """scVelo is a scalable toolkit for estimating and analyzing RNA velocities in single cells using
+ dynamical modeling."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [('hatchling', '1.18.0')]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('numba', '0.58.1'),
+    ('anndata', '0.10.5.post1'),
+    ('scanpy', '1.9.8'),
+    ('loompy', '3.0.7'),
+    ('umap-learn', '0.5.5'),
+    ('scikit-learn', '1.3.1'),
+    ('matplotlib', '3.7.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    (name, version, {
+        'modulename': 'scvelo',
+        # download from commit to not depend on scvi-tools and get rid of restriction of scikit-learn version
+        'source_urls': ['https://github.com/theislab/scvelo/archive/'],
+        'sources': [{
+            'download_filename': 'ffe2cd3c437fd385f80fed9e6abc6dc3013eff82.tar.gz',
```

please use `local_commit` here:

```suggestion
            'download_filename': '%s.tar.gz' % local_commit,
```